lib/galaxy/jobs/runners/util/cli/job/torque.py

```diff
@@ -64,7 +68,7 @@ def get_status(self, job_ids=None):
     def get_single_status(self, job_id):
         return f"qstat -f {job_id}"
 
-    def parse_status(self, status, job_ids):
+    def parse_status(self, status: str, job_ids) -> Union[Dict, None]:
```

Comment:
Wouldn't you write this as
```suggestion
    def parse_status(self, status: str, job_ids) -> Optional[Dict[str, model.Job.states]]:
```
or similar ?

Reply:
It should actually be `Dict[str, str]`, fixed in https://github.com/galaxyproject/galaxy/pull/17367/commits/237626edc0e35c76309671606dccd543dfd14c61 .

Reply:
Is there anything wrong with being more specific ? It has to be a valid job state. Happy to do that in a separate PR of course.

Reply:
Yes, when I tried it I got:
```
lib/galaxy/jobs/runners/util/cli/job/__init__.py:67: error: Variable "galaxy.jobs.runners.util.cli.job.job_states" is not valid as a type  [valid-type]
        def parse_status(self, status: str, job_ids: List[str]) -> Dict[str, job_states]:
                                                                             ^
lib/galaxy/jobs/runners/util/cli/job/__init__.py:67: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases
```

but I've solved it in https://github.com/galaxyproject/galaxy/pull/17381 .